src/components/modalDialogs/Settings.vue

```diff
@@ -30,7 +30,7 @@
             v-model="theme"
             name="theme radios"
             v-on:change="updateTheme"
-            class="settings-group"
+            class="settings-group themeTabWrapper"
```

Comment:
dunno why this is duplicated in here.... looks like you're not branched off of the latest of master.

also, i thought the classes were dash separated... must've missed this one.

Reply:
yeah, looks like you need to rebase